src/assets/scripts/client/aircraft/AircraftController.js

```diff
@@ -363,11 +366,13 @@ export default class AircraftController {
             return;
         }
 
+        const isAutoTower = GameController.getGameOption(GAME_OPTION_NAMES.TOWER_CONTROLLER) === 'SYSTEM';
```

Comment:
How would you feel about moving this whole thing to the `GameController`? So usage here would be a few lines down `GameController.isAutoTower()`. Then we'd be able to use this anywhere 😄

Reply:
Implemented `GameController.isAutoTower()`. However, from an optimization standpoint it still seems preferable to fetch that value once here and subsequently pass it to each aircraft instance as we loop through them below, than have multiple calls individually.

Reply:
See below, you're absolutely correct. Disregard what I said (sorry for the back and forth), your original implementation is way better.